src/structlog/processors.py

```diff
@@ -166,6 +168,10 @@ def __call__(
 
         return " ".join(elements)
 
+    def _escape_double_quotes(self, text: str) -> str:
+        value = f"{text}".replace(r"\"", r"\\\"")
+        return self._double_quotes_without_backslash_prefix.sub(r"\"", value)
```

Comment:
I think there's one more corner case here: if the value is between quotes, then the `\` need to be escaped.
For example, the raw string `\ "` needs to be encoded as `"\\ \"`, not `"\ \"` (confirmed on loki).